docker-images/pysyft-worker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-slim
+
+ENV WORKER_DIR /worker
+
+# Install PySyft
+RUN apt-get update && apt-get install -y gcc
+RUN pip install --no-cache-dir syft
+
+# Add worker code
+RUN mkdir $WORKER_DIR
+WORKDIR $WORKER_DIR
+COPY ./entrypoint.sh .
+RUN chmod +x ./entrypoint.sh
```

Comment:
entrypoint.sh should be made executable in git?

Reply:
I prefer to keep it as is, it's not necessary to have it as executable (no one is gonna use it) as long as we have that `chmod`